aten/src/ATen/core/dispatch/OperatorEntry.h

```diff
@@ -98,7 +99,7 @@ class TORCH_API OperatorEntry final {
   // attempt to register a schema when one is already present or vice
   // versa that is an error.  (Refcounting for the registrations is
   // handled in the OperatorHandle in Dispatcher)
-  void registerSchema(FunctionSchema&&, std::string&& debug);
+  void registerSchema(FunctionSchema&&, std::string&& debug, const c10::optional<std::unordered_set<Tags>>& tags = c10::nullopt);
```

Comment:
just fwiw, I imagine that for the number of tags we'll end up having for each operator a `std::vector` will be faster than `unordered_set`.. maybe we never expect to look up a tag on the hot path and that's fine

Reply:
yeah you are right, I have a local change which makes it a vector